parent/pom.xml

```diff
@@ -107,10 +107,10 @@
 
         <slf4j.version>1.7.6</slf4j.version>
         <jacoco.version>0.8.7</jacoco.version>
-        <graphql.client.version>1.7.6</graphql.client.version>
+        <graphql.client.version>1.7.7</graphql.client.version>
         <magento.graphql.version>9.0.0-magento242ee</magento.graphql.version>
 
-        <core.wcm.components.version>2.17.10</core.wcm.components.version>
+        <core.wcm.components.version>2.18.0</core.wcm.components.version>
         <core.wcm.components.library.version>2.9.0</core.wcm.components.library.version>
```

Comment:
There were several patch releases in the 2.18.x series. Would it worth to use a newer one?